db/schema.rb

```diff
@@ -323,6 +323,24 @@
     t.index ["unique_id", "facility_type"], name: "index_base_facilities_on_unique_id_and_facility_type", unique: true
   end
 
+  create_table "benefits_intake_form_submissions", force: :cascade do |t|
+    t.string "form_type", null: false
+    t.uuid "benefits_intake_uuid"
+    t.uuid "submitted_claim_uuid"
+    t.jsonb "form_data", default: {}
```

Comment:
this is redundant with (hopefully) what is in `saved_claims`

Reply:
Simple Forms doesn't use `saved_claims` though so I'd rather keep this in.

Reply:
Should there be a simple_forms table and model then?  Which would function much (almost exactly) like SavedClaims.
Just trying to keep the DB as normalized as possible